File: SPA/FrontEnd/src/app/Models/Region/region.ts
import { PuntoTuristico } from '../Punto/punto';
export class Region {
    nombre: string;
    puntos: PuntoTuristico[];
    id: number;

    constructor(name: string, puntos: PuntoTuristico[], id: number) {
        this.nombre = name;
        this.id = id;
        this.puntos = puntos;
    }
}

File: Domain/Region.cs
using BuisnessLogic.Domain;
using System;
using System.Collections.Generic;
using System.Text;

namespace Domain
{
    public class Region
    {
        public int Id { get; set; }
        public string Nombre { get; set; }
        public virtual List<PuntoTuristico> Puntos { get; set; }
    }
}
